### DISCUSSION TITLE
Multiple price points under the same extension

### DISCUSSION THREAD

**Discussion by akump** [Glench/ExtPay#60] [Ideas]
Hi,

Currently my pricing model is $2 a month for the extension. However, I have since learned that stripe takes a flat cut of ~30 cents and that really eats into profit especially long term. I'd like to offer a 6 month/1 year option, potentially with a discount, in addition to the $2 a month in order to combat the stripe fee on each transaction. I know ExtPay currently offers 1 year subscriptions, but I'd really prefer the flexibility for the user to choose how many months they want, 1/6/12 months. I might be able to do this by creating a second/third extension within ExtPay but then you would also need double/triple the logic in the chrome extension to handle all the different ExtPays.

**Comment by surtic86** [comment]
Same as #58
